webpack.config.dll.js

```diff
@@ -1,42 +1,46 @@
 const webpack = require('webpack');
 
-module.exports = {
-  context: process.cwd(),
+module.exports = () => {
+  const isDevMode = process.env.NODE_ENV === 'development';
 
-  devtool: 'source-map',
+  return {
+    context: process.cwd(),
+    mode: isDevMode ? 'development' : 'production',
+    devtool: isDevMode && 'cheap-module-eval-source-map',
```

Comment:
one thing we did with Calypso was to let someone specify a sourcemap from the CLI as well since we sometimes want to make sure we get a full sourcemap to debug

```js
const devTool = process.env.SOURCEMAP || ( isDevMode && 'cheap-module-eval-source-map' )
```

https://github.com/Automattic/wp-calypso/blob/e46363e640727564c989bcd33c9407674bc4623d/webpack.config.js#L124